pkg/networkservice/chains/nsmgr/utils_test.go

```diff
@@ -99,9 +101,21 @@ func newPassTroughClient(networkService string) *passThroughClient {
 }
 
 func (c *passThroughClient) Request(ctx context.Context, request *networkservice.NetworkServiceRequest, opts ...grpc.CallOption) (*networkservice.Connection, error) {
+	oldNetworkServiceEndpointName := request.Connection.NetworkServiceEndpointName
+	oldNetworkService := request.Connection.NetworkService
+
```

Comment:
Why is this changed?

Reply:
suggested by @Bolodya1997

Reply:
@Bolodya1997 Could you explain this moment?